src/headers/tomcrypt_math.h

```diff
@@ -15,6 +15,12 @@
    typedef void rsa_key;
 #endif
 
+#ifndef LTC_MILLER_RABIN_REPS
+   /* Number of repetitions of the Miller-Rabin test
```

Comment:
I would prefer to call it "rounds" instead of "repetitions".

**BUT** I think that all changes related to `is_prime`, M-R tests, missing Lucas etc. deserves a separate PR. I am not sure whether it is ready for 1.18